packages/evolution-frontend/src/components/survey/__tests__/__snapshots__/Question.test.tsx.snap

```diff
@@ -1566,140 +1566,98 @@ exports[`With help popup and link Test widget with help 1`] = `
 </div>
 `;
 
-exports[`With joining questions Joining previous 1`] = `
-Array [
-  <div
-    className="apptr__form-container"
-    style={
-      Object {
-        "marginBottom": "-40px",
-        "marginTop": "-40px",
-        "paddingBottom": "20px",
-        "paddingTop": "20px",
-      }
+exports[`With joining questions Joining next 1`] = `
+<div
+  className="apptr__form-container apptr__form-join-next two-columns question-type-string question-filled question-valid"
+  onClick={[Function]}
+  style={
+    Object {
+      "position": "relative",
     }
-  >
-    <label
-      style={
-        Object {
-          "height": "20px",
-        }
-      }
-    />
-  </div>,
+  }
+>
   <div
-    className="apptr__form-container two-columns question-type-string question-filled question-valid"
-    onClick={[Function]}
-    style={
-      Object {
-        "position": "relative",
-      }
-    }
+    className="apptr__form-label-container two-columns"
   >
-    <div
-      className="apptr__form-label-container two-columns"
-    >
-      <label
-        htmlFor="survey-question__home.region"
-      >
-        <div
-          dangerouslySetInnerHTML={
-            Object {
-              "__html": "English text",
-            }
-          }
-        />
-      </label>
-    </div>
-    <div
-      className="apptr__form-input-container two-columns"
+    <label
+      htmlFor="survey-question__home.region"
     >
-      <input
-        autoComplete="none"
-        className="apptr__form-input apptr__input-string input-large"
-        id="survey-question__home.region"
-        maxLength={255}
-        name="survey-question__home.region"
-        onBlur={[Function]}
-        onChange={[Function]}
-        style={
+      <div
+        dangerouslySetInnerHTML={
           Object {
-            "textTransform": "none",
+            "__html": "English text",
           }
         }
-        type="text"
-        value=""
       />
-    </div>
-  </div>,
-]
-`;
-
-exports[`With joining questions Joining next 1`] = `
-Array [
+    </label>
+  </div>
   <div
-    className="apptr__form-container two-columns question-type-string question-filled question-valid"
-    onClick={[Function]}
-    style={
-      Object {
-        "position": "relative",
+    className="apptr__form-input-container two-columns"
+  >
+    <input
+      autoComplete="none"
+      className="apptr__form-input apptr__input-string input-large"
+      id="survey-question__home.region"
+      maxLength={255}
+      name="survey-question__home.region"
+      onBlur={[Function]}
+      onChange={[Function]}
+      style={
+        Object {
+          "textTransform": "none",
+        }
       }
+      type="text"
+      value=""
+    />
+  </div>
+</div>
+`;
+
+exports[`With joining questions Joining previous 1`] = `
```

Comment:
Now that we only join with next, this second test is not necessary anymore

Reply:
Did you push the branch where you removed it?

Reply:
yes, should be fine now